scripts/vscode-build.ps1

```diff
@@ -15,9 +15,11 @@ if ($projectFileInfo) {
   $buildTool = InitializeBuildTool
   $frameworkArg = if ($framework -ne "") { " -p:TargetFramework=$framework" } else { "" }
   $buildArgs = "$($buildTool.Command) -v:m -m -p:UseRoslynAnalyzers=false -p:GenerateFullPaths=true$frameworkArg $($projectFileInfo.FullName)"
+  # InitializeBuildTool is hard coded for dotnet.exe, so use Ensure-DotnetSdk as it is xplat aware.
+  $buildPath = Ensure-DotnetSdk
```

Comment:
Does the 'msbuild current project skip analyzers' task still work as expected?

Reply:
I don't know, I can't test it.

Reply:
Just to be clear, I pulled this down and manually tested on Windows. 👍